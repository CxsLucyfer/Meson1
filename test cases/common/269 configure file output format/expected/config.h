/*
 * Autogenerated by the Meson build system.
 * Do not edit, your changes will be lost.
 */

#pragma once

#define bool

#undef false

/* ultimate question of life, the universe, and everything */
#define int 42

/* This is
a multiline
description */
#define str "hello world!"

#define unquoted float

