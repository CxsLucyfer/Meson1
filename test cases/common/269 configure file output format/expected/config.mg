/*
 * Autogenerated by the Meson build system.
 * Do not edit, your changes will be lost.
 */

#ifndef CONFIG_MAGNESIUM_H
#define CONFIG_MAGNESIUM_H

#define bool

#undef false

/* ultimate question of life, the universe, and everything */
#define int 42

/* This is
a multiline
description */
#define str "hello world!"

#define unquoted float

#endif
