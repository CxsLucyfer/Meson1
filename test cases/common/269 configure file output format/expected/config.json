{"bool": true, "false": false, "int": 42, "str": "\"hello world!\"", "unquoted": "float"}